### PR TITLE
Update to Bot API 6.2

### PR DESCRIPTION
**Pull request by rainrisa**

### CODE REVIEW

File: message.d.ts
Comment (dcdunkan) [inline message.d.ts:359]: Are you sure this is optional?

File: message.d.ts
Comment (dcdunkan) [inline message.d.ts:670]: In 6.2 changelogs, they have added `type` property. (Seems like you've added it in the `StickerSet` but not here)

File: proxied.d.ts
Comment (dcdunkan) [inline proxied.d.ts:1178]: They have also removed `contains_masks` (backward compatibility, yes) property from `createNewStickerSet`.

File: message.d.ts
Comment (rainrisa) [reply message.d.ts:359]: Ah, because it has its own interface, it become not optional, right?
Sorry my bad, I finally realized that

File: message.d.ts
Comment (rainrisa) [reply message.d.ts:670]: Yeah this is my mistake because I'm not careful in reading the documentation

File: proxied.d.ts
Comment (rainrisa) [reply proxied.d.ts:1178]: Again, my bad, I will bring it back!

File: message.d.ts
Comment (dcdunkan) [reply message.d.ts:359]: No. What I am trying to ask is, are you sure that updates containing `custom_emoji` message entities may not have `custom_emoji_id` property?

For example,
- `pre` entities have their `language` property optional because sometimes updates will not have `language` property in the `pre` entity.
- But `url` entities **will** have `url` property.

File: message.d.ts
Comment (dcdunkan) [reply message.d.ts:670]: Its okay, don't worry :) You have time to fix things.

File: proxied.d.ts
Comment (dcdunkan) [reply proxied.d.ts:1178]: No 👀, don't bring them back. You have to remove the `contains_masks`.

File: message.d.ts
Comment (rainrisa) [reply message.d.ts:359]: I think they should have

because `custom_emoji_id` is unique identifier of the custom emoji
So if the type of the message entity is `custom_emoji`, then there should be an `custom_emoji_id`

Comment (rainrisa) [reply message.d.ts:359]: What do you think?

File: proxied.d.ts
Comment (rainrisa) [reply proxied.d.ts:1178]: Okay 🙂

File: message.d.ts
Comment (dcdunkan) [reply message.d.ts:359]: Yes, I think so. You could do that, or wait for the review of the main maintainer of this repo (@\KnorpelSenf), and do as he suggest.

Comment (rainrisa) [reply message.d.ts:359]: Thank you very much

**Comment by KnorpelSenf** [comment]
These changes are incomplete. We try to keep the types perfectly in sync with the API.

File: manage.d.ts
Comment (KnorpelSenf) [inline manage.d.ts:89]: This is incorrect. This field is only returned in `getChat`.

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

**Comment by KnorpelSenf** [comment]
Many documentation strings are outdated, too. I will quickly fix this up :)

**Review by KnorpelSenf — APPROVED** [review]
Now this is good, thanks for the contribution!

**Pull request merged**
